onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -998,6 +1025,42 @@ public void onClick(DialogInterface dialog, int which) {
                             new DialogInterface.OnClickListener() {
                                 @Override
                                 public void onClick(DialogInterface dialog, int which) {
+                                    createZipCodeDialog().show();
+                                }
+                            }
+                    );
+            return builder.create();
+        }
+
+        private Dialog createZipCodeDialog() {
```

Comment:
Let's move this to `UIUtils`, and have it return the `Dialog` so we just show it here - it may be useful elsewhere, and `BaseMapFragment` is already pretty huge.